examples/admin/cloud_messaging.py

```diff
@@ -0,0 +1,212 @@
+#
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import datetime
+
+from firebase_admin import messaging
+
+def send_to_token():
+  # [START send_to_token]
+  # This registration token comes from the client FCM SDKs.
+  registration_token = 'YOUR_REGISTRATION_TOKEN'
+
+  # See documentation on defining a message payload.
+  message = messaging.Message(
+    data={
+      'score': '850',
+      'time': '2:45',
+    },
+    token=registration_token,
+  )
+
+  # Send a message to the device corresponding to the provided
+  # registration token.
+  response = messaging.send(message)
+  # Response is a message ID string.
+  print('Successfully sent message:', response)
+  # [END send_to_token]
+
+def send_to_topic():
+  # [START send_to_topic]
+  # The topic name can be optionally prefixed with "/topics/".
+  topic = 'highScores'
+
+  # See documentation on defining a message payload.
+  message = messaging.Message(
+    data={
+      'score': '850',
+      'time': '2:45',
+    },
+    topic=topic,
+  )
+
+  # Send a message to the devices subscribed to the provided topic.
+  response = messaging.send(message)
+  # Response is a message ID string.
+  print('Successfully sent message:', response)
+  # [END send_to_topic]
+
+def send_to_condition():
+  # [START send_to_condition]
+  # Define a condition which will send to devices which are subscribed
+  # to either the Google stock or the tech industry topics.
+  condition = "'stock-GOOG' in topics || 'industry-tech' in topics"
+
+  # See documentation on defining a message payload.
+  message = messaging.Message(
+    notification=messaging.Notification(
+      title='$GOOG up 1.43% on the day',
```

Comment:
Do you have to escape the `%` here?  Github's python parser seems to think so.

Reply:
Seems to work fine. Following works as expected in the interpreter:

```
>>> s = '$GOOG up 1.43% on the day' 
>>> print(s)
$GOOG up 1.43% on the day
```